frontend/model/contracts/group.js

```diff
@@ -183,11 +184,13 @@ DefineContract({
     'gi.contracts/group/invite': {
       validate: objectOf({
         inviteSecret: string, // NOTE: simulate the OP_KEY_* stuff for now
-        numInvites: number
+        numInvites: number,
+        creator: string // NOTE/TODO: to be used on group settings link management
```

Comment:
`creator` doesn't need to be passed in here because the same info is already available via `meta.username`

Reply:
You are right. Although I still added it to the state so @SebinSong can implement the links page management. There, we need to know who is invite creator.

Reply:
@sandrina-p if that makes it able to look up invite links based on the creator of the invite(the owner of the invite), that would be definitely very helpful for me!